samples/showcase/build.gradle

```diff
@@ -21,15 +21,25 @@ android {
 
         vectorDrawables.useSupportLibrary = true // needed for tinting
     }
+
     buildTypes {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-showcase.pro'
         }
     }
+
     flavorDimensions "main"
     productFlavors {
-        main
+        main {
+            buildTypes {
+                debug {
+                    minifyEnabled true
```

Comment:
Does this mean we're going to minify all debug builds if instant run is turned off? Could we instead have a separate "instrumentation" target?

Reply:
It's only in the main flavor, the `instantRunConfig` flavor is not affected. If main flavor is used a lot, I think we can create a new flavor just for instrumentation test.

Reply:
I think main flavour is used a bunch, since instantRun is still kinda buggy. Do you mind adding a separate buildType?

Reply:
I add a flavor called internal to do instrumentation test.